deploy/crds/operatorsource.crd.yaml

```diff
@@ -29,6 +29,14 @@ spec:
     type: string
     description: App registry namespace
     JSONPath: .spec.registryNamespace
+  - name: DisplayName
+    type: string
+    description: Display Name for resulting CatalogSource
```

Comment:
We could make this more generic as this can be used even if stopped using `CatalogSource` as our datastore.

Reply:
Does it make sense for `OperatorSource` to ever have a display name? I didn't think there was a 1:1 mapping between `OperatorSource` and anything in the ui. Would that ever change? It seems like an arbitrary abstraction from that perspective. Personally I think your argument to call all of these fields `csDisplayName` and `csPublisherName` behind the scenes since they only exist right now to pass along a value makes sense. I'm not as sold that we would need a real `DisplayName` for `OperatorSource`. Where would that be used?

Reply:
You are correct in that they wouldn't be used anywhere today. I just wanted to future proof this once `OperatorSource` makes it's way into the UI. I also think @jeff-phillips-18 is working on a UI that allows adding of `OperatorSources`. Not sure if that is for 4.0 or the future.

Reply:
Yes, we are adding a filter for the operator source type in the Operator Hub in console. Currently I am hard coding the display text for the operator source types.

Reply:
@jeff-phillips-18 are you referring to the `opsrc-provider` label the UI is using to filter `Community`, `ISV` and such?